lxqt-config-monitor/fastmenu.cpp

```diff
@@ -58,7 +58,16 @@ void FastMenu::extended()
         pos.setY(0);
         output->setPos(pos);
         output->setEnabled(true);
-        width += output->currentMode()->size().width();
+        //first left one as primary
+        output->setPrimary(width == 0);
+        KScreen::ModePtr mode(output->currentMode());
+        if (!mode) { // set first mode
```

Comment:
Please, use styling (open/close brace on new line) as it is used in the surrounding code.